### DISCUSSION TITLE
What is the new command to open the project?

### DISCUSSION THREAD

**Discussion by juantor16** [Synthetixio/synpress#538] [Q&A] [answered]
Hello, I am trying to execute the command: yarn synpress open but when I do, It fails since that command is unknown, what is the correct way of opening synpress and run the test cases like that? Thank you

**Comment by drptbl** [comment, marked as answer]
`synpress open` is not (fully) supported yet, please use `synpress run` instead.

Please, follow the readme. Everything is described there.

Thanks,
Jakub.
